code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -652,6 +652,10 @@
 	else
 		..()
 
+/obj/item/card/id/syndicate/Destroy()
+	..()
+	registered_user = null
```

Comment:
We always want `return ..()` at the end of destroy
```suggestion
/obj/item/card/id/syndicate/Destroy()
	registered_user = null
	return ..()
```

Reply:
Sorry my mind isn't fully here, fixed